lib/contentful/client.rb

```diff
@@ -296,6 +298,16 @@ def sync(options = { initial: true })
 
     private
 
+    def ensure_select_is_doable(query)
+      return unless query.key?(:select)
+
+      query[:select] = query[:select].split(',').map(&:strip) if query[:select].is_a? String
```

Comment:
I assume this is sort of a design choice, but I find it interesting that we add support for this "alternative" way of supplying a select statement, e.g. you can use a string if you feel like it or an array. Does it feel like offering this choice is consistent with how we do things elsewhere? I notice for example that the order attribute just takes a comma separated list of strings.

(I'm guessing we're coercing the values to an array here in order to do the normalization stuff below.)